services/app-api/src/resolvers/submitHealthPlanPackage.ts

```diff
@@ -258,8 +259,14 @@ export function submitHealthPlanPackageResolver(
         }
 
         // Send emails!
+        const isContractAndRate =
+            lockedFormData.submissionType === 'CONTRACT_AND_RATES'
+
         const name = packageName(lockedFormData, programs)
         const status = packageStatus(updatedPackage)
+        const rateName = isContractAndRate
```

Comment:
This kind feels like something that should go in the emailer functions not up in the resolver